ds-sklearn/description.md

```diff
@@ -0,0 +1,12 @@
+This hands-on workshop will provide you with the basics of machine learning using Python.
+
+Machine learning is the field devoted to methods and algorithms that 'learn' from data. 
+It can be applied to a vast range of different domains, from linguistics to physics and from medical imaging to history.
+
+This introduction aims to cover the basics of machine learning in a practical and hands-on manner, 
+so that upon completion, you will be able to train your first machine learning models and understand what next steps to take to improve the them.
```

Comment:
```suggestion
so that upon completion, you will be able to train your first machine learning models and understand what next steps to take to improve them.
```